### DISCUSSION TITLE
PnP Search v4 Column Resizing not working

### DISCUSSION THREAD

**Discussion by renewtx** [microsoft-search/pnp-modern-search#2237] [Q&A]
Hello,

I created a custom page in M365 and added a Search Result web part.  When I run a search I do get results

![Capture1](https://user-images.githubusercontent.com/58831734/179600628-7e6d6c20-1406-49e3-9b64-7acf3bfc6115.JPG)

I also configured all the columns in the result grid to be resizable, but I am not able to resize any of the columns.

![Capture2](https://user-images.githubusercontent.com/58831734/179603245-4ee86b5b-e3f6-4899-a5c9-f524ebd8ed87.JPG)

Is this supposed to work or am I missing something?  

On a related note, is there a way to make the web part automatically extend its width to match the minimum column widths set in the web part so that you do not have to manually either resize the columns or scroll down to get to the scroll bar and then scroll right and left?  In my screen shot above, for example, the last column (Url) is cut off and initially it is hidden because it falls outside the range of the window -- I have to first scroll to the bottom of the web part, then scroll to the right, then scroll back up to resize the column (which currently does not seem to work).

Thanks

**Comment by wobba** [comment]
@FranckyC do you know?

**Comment by kasperbolarsen** [comment]
Resize works fine in V4.18.2
